styles/stylesheet.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background-color: rgb(236,236,236);
+  align-items: center;
+  margin: 1rem;
+}
+
+.dogface {
```

Comment:
This isn't quite the right way to do this. We'll cover the proper use of background images in a later module.